R/fit.R

```diff
@@ -314,28 +314,31 @@ CmdStanFit$set("public", name = "init", value = init)
 #' @aliases init_model_methods
 #'
 #' @description The `$init_model_methods()` method compiles and initializes the
-#'   `log_prob`, `grad_log_prob`, `constrain_variables`, `unconstrain_variables`
-#'   and `unconstrain_draws` functions. These are then available as methods of
-#'   the fitted model object. This requires the additional `Rcpp` package,
-#'   which are not required for fitting models using
+#'   `log_prob`, `grad_log_prob`, `hessian`, `constrain_variables`,
+#'   `unconstrain_variables` and `unconstrain_draws` functions. These are then
+#'   available as methods of the fitted model object. This requires the
+#'   additional `Rcpp` package, which are not required for fitting models using
 #'   CmdStanR.
 #'
-#'   Note: there may be many compiler warnings emitted during compilation but
+#'   Notes:
+#'   * There may be many compiler warnings emitted during compilation but
 #'   these can be ignored so long as they are warnings and not errors.
+#'   * The hessian method relies on higher-order autodiff, which is still
+#'   experimental. Please report any compilation errors that you encounter.
```

Comment:
This comment can go - I've actually changed the `hessian` code over to use finite-diffs rather than autodiff (consistent with bridgestan and newton optimisation in cmdstan), because there's still a bunch of functions that aren't autodiff compatible (sad face) 

Completely missed removing this argument/message somehow - sorry!

Reply:
No problem, I’ll remove it.